pontoon/base/static/css/heading_info.css

```diff
@@ -45,17 +45,22 @@ ul {
 }
 
 #heading .details .value {
+  color: #AAA;
   float: right;
 }
 
 #heading .details .value a {
-  color: #EBEBEB;
+  color: #FFFFFF;
 }
 
 #heading .details .value a:hover {
   color: #7BC876;
 }
 
+#heading .details li:hover .value a {
+  color: #7BC876;
+}
+
```

Comment:
I'd remove this, too. Because when you hover over "REPOSITORY" on http://localhost:8000/en-GB/pontoon-intro/, the URL changes to green, but you can't click (because "REPOSITORY" is not a link).

Reply:
@mathjazz  the repository seems to be link I even clicked it and it lead me here https://github.com/mozilla/pontoon-intro

Reply:
Yup I have also checked it on mozilla.pontoon.org and it indeed is a link

Reply:
Sure it is. But the title "REPOSITORY" is not. And it turns green on hover. So you'd expect it to be link, too.

Reply:
only links are green may be some confusion repository is not turning green at all. I will update all the screenshots with next commit so that it solves all the confusions. :)

Reply:
@mathjazz hope this one gets everything right sorry for the blunders but a link or two always tend to miss out

Reply:
screenshots updated!

Reply:
> only links are green may be some confusion repository is not turning green at all.

This patch makes all the links white by default and green on hover, and all non-links grey, consistently across the entire dashboard heading. Making some of the non-links green on hover breaks that consistency. As I already said, it's also misleading - because you expect such text to be a link after you hover.

Please remove this rule and we're good to go.